userguide/aws/how-to-maintain-a-multi-tier-system.adoc

```diff
@@ -8,34 +8,21 @@ This guide explains how to maintain your multi-tier Kill Bill installation on AW
 
 To perform most maintenance tasks you will need to login to your EC2 instances. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide.
 
```

Comment:
> Login procedures are described in your setup guide.

Would be good to add a link to the corresponding setup guide.

Reply:
I agree. I will add the link.